gamemodes/terrortown/gamemode/server/sv_player.lua

```diff
@@ -382,7 +385,7 @@ end
 -- @ref https://wiki.facepunch.com/gmod/GM:PlayerUse
 -- @local
 function GM:PlayerUse(ply, ent)
-    return ply:IsTerror()
+    return overrideDoPlayerUse
```

Comment:
what's the reason here for having this in a variable?

Reply:
Ok, I get it now. But I would do it differently. I'd add a third parameter to `GM:PlayerUse(ply, ent, overrideDoPlayerUse)`

And then when you hook.Run: `Hook.Run("PlayerUse", ply, ent, true)`

I think that should handle all cases the way you want without using a local global variable.